vscode/src/prompt-builder/index.test.ts

```diff
@@ -373,10 +373,9 @@ describe('PromptBuilder', () => {
                 .join('\n')
 
             expect(builder.contextItems.length).toBe(1)
-            const filePath = convertFileNameByPlatform('foo/bar.go')
             expect(promptContent).toMatchInlineSnapshot(`
               "preamble
-              Codebase context from file ${filePath}:
+              Codebase context from file ${PromptString.fromDisplayPath(file.uri)}:
```

Comment:
```suggestion
              Codebase context from file ${displayPath(file.uri)}:
```
nit: I think we can use the `displayPath` function directly without turning it into a PromptString. Feel free to ignore this 😄